glean-core/src/upload/request.rs

```diff
@@ -262,4 +273,15 @@ mod test {
         assert!(request.headers.contains_key("X-Client-Version"));
         assert!(request.headers.contains_key("Content-Length"));
     }
+
+    #[test]
+    fn errors_when_request_body_exceeds_max_size() {
+        let request = Builder::new(/* language_binding_name */ "Rust", 0)
```

Comment:
`0` here doesn't say much. Often `0` is also a special value to disable stuff.

Let's document that this sets an arbitrarily small value (and maybe use 1? Just a thought, don't care much about the exact value)